From 1f20563bdc3e3e4d3cd6124fbe170aaabb407a64 Mon Sep 17 00:00:00 2001
From: Chris Wang <Chris_Wang@wiwynn.com>
Date: Wed, 20 Dec 2023 16:16:06 +0800
Subject: [PATCH 1003/1024] remove pincontrol on GPIO U5

Remove pincontrol on GPIO U5 to avoid kernel occupation, let programs at
application layer can monitor the pin.

This commit depends on the patch below:
[1] https://lore.kernel.org/lkml/20240109072053.3980855-17-Delphine_CC_Chiu@wiwynn.com/
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
index 574c4d730471..3d12a628fe09 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
@@ -1454,8 +1454,7 @@ &sgpiom0 {
 &gpio0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_gpiu2_default &pinctrl_gpiu3_default
-		     &pinctrl_gpiu4_default &pinctrl_gpiu5_default
-		     &pinctrl_gpiu6_default>;
+		     &pinctrl_gpiu4_default &pinctrl_gpiu6_default>;
 	gpio-line-names =
 	/*A0-A7*/       "","","","","","","","",
 	/*B0-B7*/       "FLT_HSC_SERVER_SLOT8_N","AC_ON_OFF_BTN_CPLD_SLOT5_N",
-- 
2.44.2

